docs/csharp/language-reference/compiler-options/errors-warnings.md

```diff
@@ -53,6 +53,10 @@ To get information about an error or warning, you can look up the error code in
 
 ## Analysis level
 
+```xml
```

Comment:
```suggestion
The **AnalysisLevel** option specifies additional [warning waves](../compiler-messages/warning-waves.md) to enable. Warning wave warnings are additional checks that improve your code, or ensure it will be compatible with upcoming releases.

```xml
```

Reply:
LGTM but I'm not an SME here. I think `AnalysisLevel` might also control which analyzers are enabled too.

Reply:
cc @jmarolf @sharwell @mavasani

Reply:
> LGTM but I'm not an SME here. I think `AnalysisLevel` might also control which analyzers are enabled too.

Yes, this is correct. This is documented at https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#analysislevel

Reply:
Hi @drewnoakes I updated the suggestted based on @mavasani 's comments.

Reply:
Thanks! LGTM